capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -28,8 +28,13 @@ class BookSearchTile extends Component {
 
   markUnread = () => {
     fetch(`/api/books?id=${this.state.bookObject.id}`, {method: 'delete'})
-        .then(this.setState({hasRead: false, bookObject: {}}))
-        .catch(e => console.log(e));
+        .then(() => this.setState({hasRead: false, bookObject: {}}))
```

Comment:
nice job removing "optimistic behavior" in your promise chaining!

Reply:
Thanks! I realized this when I was going back to change this
I didn't know that was the term for it!